lib/EarlyFunctionAnalysis/DetectABI.cpp

```diff
@@ -488,6 +494,100 @@ void DetectABI::interproceduralPropagation() {
   }
 }
 
+void DetectABI::propagatePrototypes() {
+  for (model::Function &Function : Binary->Functions()) {
+    const auto &Entry = Function.Entry();
```

Comment:
Please drastically reduce the usage of `auto` ([B.15](https://pad.rev.ng/s/SJNUet_ku#))